homeassistant/components/smlight/update.py

```diff
@@ -193,5 +193,8 @@ async def async_install(
             # block until update finished event received
             await self._finished_event.wait()
 
+            # allow time for SLZB-06 to reboot before updating coordinator data
+            await asyncio.sleep(10)
```

Comment:
This smells a bit, wouldn't it be better if there was actually a try-retry logic here to actually await for it to be back instead of 10 second wait?

Reply:
Just to be clear, there is a small window (~1sec) between receiving "update complete" event and the start of reboot, that is the problem. If the call to `coordinator.async_refresh()` comes during reboot it will still succeed, although that itself probably has a 60sec timeout. I will add retry instead.